### PR TITLE
www.iobit.com

### PR DESCRIPTION
**Pull request by gork7777**
issue: #100492
`popout` is better cause it'll be block main page gift free popup

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -113,6 +113,7 @@ sodapdf.com##.be-tooltip
 ! Specific sites
 !######### Subscription popups add to subscriptions.txt ######### 
 !
+iobit.com##.popout
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:116]: Can you reproduce it? I don't see it on my end.

Comment (gork7777) [reply AnnoyancesFilter/sections/popups.txt:116]: Yes. Here: 
image

Comment (zloyden) [reply AnnoyancesFilter/sections/popups.txt:116]: Ok. it looks like an ad. What's the end destination? If it leads to another website, please do next.
1. Move your rule to the `Base - specific`.
2. Remove this `iobit.com###itoppopout`.

Comment (gork7777) [reply AnnoyancesFilter/sections/popups.txt:116]: No it doesn't leads to another website, it leads here: https://www.iobit.com/en/driver-booster.php?from=dbsptest

**Review by zloyden — APPROVED** [review]

**Pull request merged**